### PR TITLE
Add note about PL logs

### PR DESCRIPTION
**Pull request by hfaivre**
### What does this PR do?
Add a small note to the Synthetics Private Locations configuration page (en/fr/jp) to explain where logs are being output.

### Motivation
Customer enquiries, zendesk tickets.

### Preview
https://docs-staging.datadoghq.com/homere.faivre/private_locations/synthetics/private_locations/configuration/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/private_locations/configuration.md [hunk 1]
```diff
@@ -90,6 +90,8 @@ You can upload custom root certificates to your private locations to have your A
 | `dumpFullConfig` | Boolean | `none` | Display full worker configuration parameters. |
 | `help` | Boolean | `none` | Show help. |
 
+**Note**: Private Location containers output logs to stdout/stderr without saving them within the container.
```
Comment (MargotLepizzera) [inline content/en/synthetics/private_locations/configuration.md:93]:
```suggestion
**Note**: Private Location containers output logs to `stdout`/`stderr` without saving them within the container.
```

File: content/fr/synthetics/private_locations/configuration.md [hunk 2]
```diff
@@ -89,6 +89,9 @@ Vous pouvez importer des certificats racine privés dans vos emplacements privé
 | `dumpFullConfig` | Booléen | `none` | Afficher les paramètres de configuration du worker complets. |
 | `help` | Booléen | `none` | Afficher les informations d'aide. |
 
+**Remarque** : Pour les conteneurs des Emplacements Privés, les logs sortants sont écrits sur stdout/stderr et ne sont pas sauvegardés au sein des conteneurs.
```
Comment (MargotLepizzera) [inline content/fr/synthetics/private_locations/configuration.md:92]:
```suggestion
**Remarque** : Pour les conteneurs des Emplacements Privés, les logs sortants sont écrits sur `stdout`/`stderr` et ne sont pas sauvegardés au sein des conteneurs. 
```

File: content/ja/synthetics/private_locations/configuration.md [hunk 3]
```diff
@@ -89,6 +89,8 @@ docker run --rm -v $PWD/<MY_WORKER_CONFIG_FILE_NAME>.json:/etc/datadog/synthetic
 | `dumpFullConfig` | Boolean | `none` | 完全なワーカーコンフィギュレーションパラメーターを表示します。 |
 | `help` | Boolean | `none` | ヘルプを表示します。 |
 
+**注**: プライベートロケーションコンテナログはコンテナ内に保存されず、stdout/stderrに出力されます。
```
Comment (MargotLepizzera) [inline content/ja/synthetics/private_locations/configuration.md:92]:
```suggestion
**注**: プライベートロケーションコンテナログはコンテナ内に保存されず、`stdout`/`stderr`に出力されます。
```

**Comment by MargotLepizzera** [comment]
LGTM - thanks Homère!!

**Comment by sarina-dd** [comment]
LGTM! Note: Removing the FR/JA files as they will be overwritten with our next build.

**Review by sarina-dd — APPROVED** [review]

**Pull request merged**